requirements.txt

```diff
@@ -19,7 +19,7 @@ itsdangerous==2.0.1
 Jinja2==3.0.3
 jmespath==0.10.0
 joblib==1.1.0
-kubernetes==23.6.0
+kubernetes==24.2.0
```

Comment:
24.3?

Reply:
No, for whatever reason the python client libs don't line up the same
https://pypi.org/project/kubernetes/